api/src/setup/index.ts

```diff
@@ -1,113 +1,204 @@
+            }
+        }
+        catch (e) {
+            logger.error(e.message)
+        }
+        
+        logger.log('Create identity...');
+    
+        const serverData: CreateIdentityBody = serverIdentityJson;
+        
+        const ssiService = SsiService.getInstance(this.identityConfig, logger);
+        const userService = new UserService(ssiService, this.serverSecret, logger);
+        const identity = await userService.createIdentity(serverData);
+    
+        logger.log('==================================================================================================');
+        logger.log(`== Store this identity in the as ENV var: ${identity.doc.id} ==`);
+        logger.log('==================================================================================================');
+    
+        // logger.log(JSON.stringify(identity, null, 2))
+    
+        // re-create the verification service with a valid server identity id
+        const verificationService = new VerificationService(
+            ssiService,
+            userService,
+            {
+                serverSecret: this.serverSecret,
+                serverIdentityId: identity.doc.id,
+                keyCollectionSize: KEY_COLLECTION_SIZE
+            },
+            logger
+        );
+    
+        const serverUser = await userService.getUser(identity.doc.id);
 
-		logger.log('Generate key collection...');
-		const index = await VerifiableCredentialsDb.getNextCredentialIndex(this.serverIdentityId);
-		const keyCollectionIndex = verificationService.getKeyCollectionIndex(index);
-		const kc = await verificationService.getKeyCollection(keyCollectionIndex);
+        if (!serverUser) {
+            throw new Error('server user not found!');
+        }
+        logger.log('Add server id as trusted root...');
+        await addTrustedRootId(serverUser.identityId);
+    
+        logger.log('Generate key collection...');
+        const index = await VerifiableCredentialsDb.getNextCredentialIndex(serverUser.identityId);
+        const keyCollectionIndex = verificationService.getKeyCollectionIndex(index);
+        const kc = await verificationService.getKeyCollection(keyCollectionIndex);
+    
+        if (!kc) {
+            throw new Error('could not create the keycollection!');
+        }
+    
+        logger.log('Set server identity as verified...');
+        const subject: Subject = {
+            claim: serverUser.claim,
+            credentialType: CredentialTypes.VerifiedIdentityCredential,
+            identityId: serverUser.identityId
+        };
 
-		if (!kc) {
-			throw new Error('could not create the keycollection!');
-		}
+        await verificationService.verifyIdentity(subject, serverUser.identityId, serverUser.identityId);
+        
+        writeFileSync(this.serverIdentityId, JSON.stringify({
+            root: serverUser.identityId,
+            identity: identity.doc
```

Comment:
is it really necessary to store the identity doc here? if so please name it `doc` instead of `identity` this is more convinient, if not please remove it